LongWarOfTheChosen/Src/LW_Toolbox_Integrated/Classes/XComGameState_Unit_LWRandomizedStats.uc

```diff
@@ -99,26 +99,32 @@ function RandomizeInitialStats(XComGameState_Unit Unit)
 		ComIntTotalWeight += ComIntSwap.Weight;
 	}
 
+	//Roll a chance for a stat swap and don't roll combat intelligence for rebels
+	if (`SYNC_RAND(100) < default.COMINT_BASE_SWAP_CHANCE && InStr(Unit.GetMyTemplateName(), "Rebel") != 0 )
+	{
+		do 
+		{
+			ComIntSwap = SelectComIntStatSwap(ComIntTotalWeight);
+		} until (IsValidComIntSwap(ComIntSwap, Unit) || (++iterations > 1000));
+		if (iterations <= 1000)
+		{
+			CharacterInitialStats_Deltas[ComIntSwap.StatSwapped] += ComIntSwap.StatSwapped_Amount;
+			CharacterComIntDelta = ComIntSwap.ComIntStatchange;	
+		}
+	}
 	//randomly apply a bunch of stat swaps to get starting stat offset
 	for(idx = 0; idx < NumSwaps; idx++)
 	{
 		do {
 			Swap = SelectRandomStatSwap(TotalWeight);
 		} until (IsValidSwap(Swap, Unit) || (++iterations > 1000));
-
+		if (iterations > 1000)
```

Comment:
Considering the number of times `1000` is used, I think it's about time we assigned it to a constant, say `MAX_SWAP_ATTEMPTS`.